integration_test/minikube_adm/minikube_adm.go

```diff
@@ -0,0 +1,214 @@
+package minikubeadm
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	. "github.com/openebs/node-disk-manager/integration_test/common"
+	"github.com/openebs/node-disk-manager/pkg/util"
+)
+
+var (
+	useSudo     = true // Default value to use sudo
+	execCommand = ExecCommandWithSudo
+	runCommand  = RunCommandWithSudo
+)
+
+func init() {
+	useSudoEnv := strings.ToLower(strings.TrimSpace(os.Getenv("USE_SUDO")))
+	if useSudoEnv == "true" { // If it is mensioned in the environment variable to use sudo
+		useSudo = true // use sudo then
+	} else if useSudoEnv == "false" { // Else if it is mensioned in the environment variable not to use sudo
```

Comment:
same here ("mensioned should be mentioned")

Reply:
Addressed.